source/declarations.tex

```diff
@@ -1923,25 +1923,28 @@
 are determined as follows:
 \begin{itemize}
 \item
-for a non-discarded \tcode{return} statement that occurs
+For a non-discarded \tcode{return} statement that occurs
 in a function declared with a return type
 that contains a placeholder type,
 \tcode{T} is the declared return type
 and $E$ is the operand of the \tcode{return} statement.
```

Comment:
This doesn't work either, if we assume a syntactic translation: `return a, b;` doesn't produce an *initializer-clause* that can be the argument in a function call.

Reply:
fixed